es-vue-base/src/lib-components/EsFileInput.vue

```diff
@@ -206,6 +206,10 @@ export default {
             this.pickedItems = [];
         },
         async verifyMimeType(file) {
+            // If a folder then trigger fileTypeError
+            if (file.type === '') {
+                this.$emit('fileIsAFolderError');
```

Comment:
Can you add a listener in the ES-DS page to showcase this? https://github.com/EnergySage/es-ds/blob/main/es-design-system/pages/molecules/es-file-input.vue